tools/odbc/driver.cpp

```diff
@@ -71,8 +71,8 @@ SQLRETURN SQLSetEnvAttr(SQLHENV environment_handle, SQLINTEGER attribute, SQLPOI
 	}
 	switch (attribute) {
 	case SQL_ATTR_ODBC_VERSION: {
-		auto version = (SQLINTEGER)(uintptr_t)value_ptr;
 		// TODO actually do something with this?
+		// auto version = (SQLINTEGER)(uintptr_t)value_ptr;
 		return SQL_SUCCESS;
```

Comment:
Perhaps we can add a property in `duckdb::OdbcHandleEnv` to store the version and assign here:

`env->odbc_version = (SQLINTEGER)value_ptr;`

Reply:
Not sure why this needs to happen in this PR, the version was simply unused before.

Reply:
yeah, it is not essential for this PR. (I was wondering about future usage by the driver.)